clap_complete/tests/testsuite/dynamic.rs

```diff
@@ -132,6 +133,88 @@ goodbye-world"
     );
 }
 
+#[test]
+fn suggest_argument_value() {
+    let mut cmd = Command::new("dynamic")
+        .arg(
+            clap::Arg::new("input")
+                .long("input")
+                .short('i')
+                .value_hint(clap::ValueHint::FilePath),
+        )
+        .arg(
+            clap::Arg::new("format")
+                .long("format")
+                .short('F')
+                .value_parser(["json", "yaml", "toml"]),
+        )
+        .args_conflicts_with_subcommands(true);
+
+    let testdir = snapbox::dir::DirRoot::mutable_temp().unwrap();
+    let testdir_path = testdir.path().unwrap();
+
+    fs::write(testdir_path.join("a_file"), "").unwrap();
+    fs::write(testdir_path.join("b_file"), "").unwrap();
+    fs::create_dir_all(testdir_path.join("c_dir")).unwrap();
+    fs::create_dir_all(testdir_path.join("d_dir")).unwrap();
+
+    assert_data_eq!(
+        complete!(cmd, "--input [TAB]", current_dir = Some(testdir_path)),
+        snapbox::str![
+            "a_file
+b_file
+c_dir/
+d_dir/"
+        ],
+    );
+
+    assert_data_eq!(
+        complete!(cmd, "-i [TAB]", current_dir = Some(testdir_path)),
+        snapbox::str![
+            "a_file
+b_file
+c_dir/
+d_dir/"
+        ],
+    );
+
+    assert_data_eq!(
+        complete!(cmd, "--input a[TAB]", current_dir = Some(testdir_path)),
+        snapbox::str!["a_file"],
+    );
+
+    assert_data_eq!(
+        complete!(cmd, "-i b[TAB]", current_dir = Some(testdir_path)),
+        snapbox::str!["b_file"],
+    );
+
+    assert_data_eq!(
+        complete!(cmd, "--format [TAB]"),
+        snapbox::str![
+            "json
+yaml
+toml"
+        ],
+    );
+
+    assert_data_eq!(
+        complete!(cmd, "-F [TAB]"),
+        snapbox::str![
+            "json
+yaml
+toml"
+        ],
+    );
+
+    assert_data_eq!(complete!(cmd, "--format j[TAB]"), snapbox::str!["json"],);
+
```

Comment:
This commit looks like the tests aren't passing for it.  Please update it so the tests pass and the actual feature commit updates the tests to this state.